homeassistant/components/withings/sensor.py

```diff
@@ -240,7 +240,7 @@ class WithingsSensorEntityDescription(
         key=Measurement.SLEEP_HEART_RATE_MAX.value,
         measurement=Measurement.SLEEP_HEART_RATE_MAX,
         measure_type=GetSleepSummaryField.HR_MAX,
-        translation_key="fat_mass",
+        translation_key="maximum_heart_rate",
         name="Maximum heart rate",
```

Comment:
Why is here a name too?

Reply:
Ah, I am guessing i forgot this whole item